eclair-core/src/test/scala/fr/acinq/eclair/channel/states/a/WaitForAcceptChannelStateSpec.scala

```diff
@@ -176,6 +176,31 @@ class WaitForAcceptChannelStateSpec extends TestKitBaseClass with FixtureAnyFunS
     awaitCond(alice.stateName == WAIT_FOR_FUNDING_INTERNAL)
   }
 
+  test("recv AcceptChannel (upfront shutdown script)", Tag(StateTestsTags.OptionUpfrontShutdownScript)) { f =>
+    import f._
+    val accept = bob2alice.expectMsgType[AcceptChannel]
```

Comment:
Why did you remove the checks on the message tlv? I think they are still useful, because right now if no script is sent (`None`) your test will pass.

I would add:
```suggestion
    val accept = bob2alice.expectMsgType[AcceptChannel]
    assert(accept.upfrontShutdownScript_opt === Some(Bob.channelParams.defaultFinalScriptPubKey))
```

Reply:
done in https://github.com/ACINQ/eclair/pull/1846/commits/e3174057030eb52428df1f77937f56568db4dfe4